# Total planning attempts
planning_attempts: 12

# Planning group and link names
arm_group_name: "panda_arm"
eef_name: "hand"
hand_group_name: "hand"
hand_frame: "panda_link8"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_home_pose: "ready"

# Scene frames
world_frame: "world"

# Collision object for picking CYLINDER object specifications
# object_name: "object"
object_name: "cylinder"
object_dimensions: [0.25, 0.02] # [height, radius]
object_pose: [0.3 , -0.45, 0.0, 0, 0, 0]
object_reference_frame: "world"

object2_name: "cylinder2"
object2_dimensions: [0.25, 0.02] # [height, radius]
object2_pose: [0.5, -0.25, 0.0, 0, 0, 0]
object2_reference_frame: "world"

# Table model
spawn_table: true
table_name: "table"
table_dimensions: [0.6, 0.5, 0.1]  # [length, width, height]
table_pose: [0.5, -0.25, 0, 0, 0, 0]
table_reference_frame: "world"
# surface_link: "{{ table_name }}"


table2_name: "table2"
table2_dimensions: [0.6, 0.5, 0.1]  # [length, width, height]
table2_pose: [-0.5, 0.25, 0, 0, 0, 0]
table2_reference_frame: "world"
# surface2_link: "{{ table2_name }}"

# Gripper grasp frame transform [x,y,z,r,p,y]
grasp_frame_transform: [0, 0, 0.1, 1.571, 0.785, 1.571]

# Place pose [x,y,z,r,p,y]
# place_pose: [0.6, -0.15, 0, 0, 0, 0]
# place_pose: [0.65, -0.05, 0, 0, 0, 0]
place_pose: [-0.5, 0.25, 0, 0, 0, 0]
# place_pose: [-0.65, 0.05, 0, 0, 1.571, 1.571]
# place_pose2: [-0.65, 0.05, 0, 0, 0, 0] 
place_pose2: [-0.5, 0.25, 0, 0, 0, 0]
place_surface_offset: 0.0001 # place offset from table
# place_surface_offset: 0.00001 # to place inside table

# Valid distance range when approaching an object for picking
# approach_object_min_dist: 0.1
# approach_object_max_dist: 0.15
approach_object_min_dist: 0.01
approach_object_max_dist: 0.05

# Valid height range when lifting an object after pick
# lift_object_min_dist: 0.01
# lift_object_max_dist: 0.1
lift_object_min_dist: 0.2
lift_object_max_dist: 0.4
